vng/metadata.go

```diff
@@ -265,6 +266,68 @@ func (d *Dynamic) Len() uint32 {
 	return d.Length
 }
 
+func ZoneMapValues(zctx *super.Context, m Metadata) []super.Value {
```

Comment:
There's no zone map here so I think MetadataValues would be a better name.

Reply:
I mean they are [zone maps](https://docs.oracle.com/en/database/oracle/oracle-database/21/dwhsg/using-zone-maps.html) but I'm ok with `MetadataValues`